common/src/androidTest/java/com/microsoft/identity/common/adal/internal/cache/StorageHelperTests.java

```diff
@@ -186,6 +187,31 @@ public void testEncryptSameText() throws GeneralSecurityException, IOException {
         Log.d(TAG, "Finished testEncryptSameText");
     }
 
+    @Test
+    public void testKeyThumbprint() throws GeneralSecurityException, IOException {
+        Context context = getInstrumentation().getTargetContext();
+        final StorageHelper storageHelper = new StorageHelper(context);
+        for (int i = 0; i < 1000; i++) {
+            Assert.assertEquals(storageHelper.testThumbprint(), storageHelper.testThumbprint());
```

Comment:
would it be better if we checked with here LAST_KNOWN_THUMBPRINT, instead of 2 function calls?

Reply:
It shouldn't change.  I don't really need the loop, even.

Reply:
yeah I asked because of the looping

Reply:
testThumbprint doesn't actually mess with that.

> Can we add a PR description to this? Don't have context for this work

done.  I put you on this mainly because it's messing very lightly with encryption algorithms, and though there doesn't seem to be a better way to do this given the java SecretKey api, you might know one.